Makefile

```diff
@@ -6,8 +6,9 @@ check:
 check-pypi-packaging:
 	podman run --rm -ti -v $(CURDIR):/src -w /src registry.fedoraproject.org/fedora:29 bash -c '\
 		pip3 install . \
-		&& ab --help \
-		&& ab build --help'
+		&& ansible-bender --help \
+		&& ansible-bender build --help' \
+		&& pip3 freeze | grep ansible-bender
```

Comment:
Why not have `--version` option?

Reply:
now I get your question: I just want to be sure that pip sees the package as installed